README.md

```diff
@@ -1,2 +1,23 @@
-# apim-km-auth0
-Repository to Auth0 contain api manager Auth0 implementation.
+# Integrate Auth0 as a Third Party Key Manager for WSO2 API Manager
+
+This Auth0 key manager implementation allows you to integrate the WSO2 API Developer Portal with an external Identity and Access Management server (IAM)
+by using the Auth0 OAuth Authorization Server to manage the OAuth clients and tokens required
+by WSO2 API Manager. We have a sample client implementation that consumes the APIs exposed by Auth0.
+
+## Getting Started
+
+To get started, go to [Integrate WSO2 API Manager 3.2.0 with Auth0](docs/config.md).
```

Comment:
We don't need the version here, do we?

Reply:
Will alter this and, In the details description will specify this is supported APIM 3.2.0 onwards WDYT?

Reply:
yes, better to mention that it is supported since 3.2.0

Reply:
Changed